ios/FluentUI/ActivityIndicator/ActivityIndicator.swift

```diff
@@ -59,9 +59,33 @@ public struct ActivityIndicator: View, TokenizedControlView {
                 "Accessibility.ActivityIndicator.Stopped.label".localized
         }()
 
-        SemiRing(color: color,
-                 thickness: tokenSet[.thickness].float,
-                 accessibilityLabel: accessibilityLabel)
+#if DEBUG
```

Comment:
Is there a reason why the following is in the actual control file? I'm not very well versed in xcuitests but this doesn't feel quite right to me

Reply:
We wanted accessibility identifiers to only be available in debug builds but I'm not sure if this is the right way to do it.